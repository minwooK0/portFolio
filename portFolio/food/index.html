<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>국가식품 클라스터</title>

    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="./css/index.css">
    <link rel="stylesheet" href="./css/swiper.css">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
    <link href="https://unpkg.com/aos@2.3.1/dist/aos.css" rel="stylesheet">
    <script src="https://unpkg.com/aos@2.3.1/dist/aos.js"></script>
    <script src="https://unpkg.com/swiper/swiper-bundle.js"></script>
    <script src="https://unpkg.com/swiper/swiper-bundle.min.js"></script>
    <script src="https://code.jquery.com/jquery-1.11.3.min.js"></script>
    <script type="text/javascript" src="js/header.js"></script>
    <script type="text/javascript" src="js/index.js"></script>

    <script>
        $(document).ready(function() {
            AOS.init();
        });

    </script>

</head>

<body>
    <div class="header">
        <div class="inner">
            <div class="logo"><a href="./index.html"><img src="img/main/logo.png" alt="로고이미지"></a></div>
            <div class="menu_wrap">
                <ul class="menu">
                    <li><a href="./sub1.html">국가식품클러스터</a></li>
                    <li>
                        <a href="./sub2_1.html"><span>행사소개</span></a>
                        <p class="depth_2">
                            <a href="./sub2_1.html">푸드어택 기획전 소개</a>
                            <a href="./sub2_2.html">지난 익산식품대전</a>
                        </p>
                    </li>
                    <li>
                        <a href="./sub3.html"><span>푸드어택 기획전</span></a>
                        <p class="depth_2">
                            <a href="./sub3.html">참가기업 제품소개</a>
                        </p>
                    </li>
                    <li>
                        <a href="./sub4_1.html"><span>랜선 푸드어택</span></a>
                        <p class="depth_2">
                            <a href="./sub4_1.html">푸드어택 맛 보기</a>
                            <a href="./sub4_2.html">푸드어택 리뷰</a>
                        </p>
                    </li>
                    <li>
                        <a href="./sub5.html">이벤트</a>
                    </li>
                </ul>
            </div>
            <div class="menu_info">
                <div class="menu_ct_01"><a href="https://www.foodpolis.kr/">한국식품산업클러스터진흥원</a></div>
                <div class="menu_sns">
                    <a href="https://www.facebook.com/foodpolis.kr">
                        <!-- 페이스북 -->
                    </a>
                    <a href="https://www.instagram.com/foodpolis.kr/">
                        <!-- 인스타 -->
                    </a>
                    <a href="https://blog.naver.com/foodpolis1">
                        <!-- 네이버블로그 -->
                    </a>
                    <a href="https://www.youtube.com/channel/UCcdXmMG59sjCxLyINIlTW2Q/featured">
                        <!-- 유튜브 -->
                    </a>
                </div>
                <div class="all_menu_btn win"></div>
                <div class="all_menu_btn mo"></div>
            </div>
        </div>
    </div>
    <div class="header all_menu_head">
        <div class="inner">
            <div class="logo"><a href="./index.html"><img src="img/main/logo.png" alt="로고이미지"></a></div>
            <div class="menu_info">
                <div class="menu_ct_01"><a href="https://www.foodpolis.kr/">한국식품산업클러스터진흥원</a></div>
                <div class="menu_sns">
                    <a href="https://www.facebook.com/foodpolis.kr">
                        <!-- 페이스북 -->
                    </a>
                    <a href="https://www.instagram.com/foodpolis.kr/">
                        <!-- 인스타 -->
                    </a>
                    <a href="https://blog.naver.com/foodpolis1">
                        <!-- 네이버블로그 -->
                    </a>
                    <a href="https://www.youtube.com/channel/UCcdXmMG59sjCxLyINIlTW2Q/featured">
                        <!-- 유튜브 -->
                    </a>
                </div>
                <div class="close_btn"></div>
            </div>
        </div>
        <div class="all_menu_content">
            <div class="text_ct">
                <dl>
                    <dt><a href="./sub1.html">국가식품클러스터</a></dt>
                    <dd>
                        <a href="./sub1.html">국가식품클러스터 온라인 홍보관</a>
                    </dd>
                </dl>
                <dl>
                    <dt><a href="./sub2_1.html">행사소개</a></dt>
                    <dd>
                        <a href="./sub2_1.html">푸드어택 기획전 소개</a>
                        <em>·</em>
                        <a href="./sub2_2.html">지난 익산식품대전</a>
                    </dd>
                </dl>
                <dl>
                    <dt><a href="./sub3.html">푸드어택 기획전</a></dt>
                    <dd>
                        <a href="./sub3.html">참가기업 제품소개</a>
                    </dd>
                </dl>
                <dl>
                    <dt><a href="./sub4_1.html">랜선 푸드어택</a></dt>
                    <dd>
                        <a href="./sub4_1.html">푸드어택 맛 보기</a>
                        <em>·</em>
                        <a href="./sub4_2.html">푸드어택 리뷰</a>
                    </dd>
                </dl>
                <dl>
                    <dt><a href="./sub5.html">이벤트</a></dt>
                    <dd>
                    </dd>
                </dl>
            </div>
        </div>
        <div class="all_menu_ft">
            (54576) 전라북도 익산시 왕궁면 국가식품로 100<br>
            한국식품산업클러스터진흥원 연락처 : 063-720-0500<br>
            Copyright(c) 한국식품산업클러스터진흥원 All rights reserved.
        </div>
    </div>
    <div class="mobile_menu_wrap">
        <div class="mobile_menu">
            <div class="menu_top">
                <div class="logo"><a href="./index.html"><img src="img/main/logo.png" alt="로고이미지"></a></div>
                <div class="close_btn"></div>
            </div>
            <div class="menu_bottom">
                <div class="menu">
                    <dl>
                        <dt><a href="./sub1.html">국가식품클러스터</a></dt>
                        <dd>
                            <a href="./sub1.html">국가식품클러스터 온라인 홍보관</a>
                        </dd>
                    </dl>
                    <dl>
                        <dt><a href="./sub2_1.html">행사소개</a></dt>
                        <dd>
                            <a href="./sub2_1.html">푸드어택 기획전 소개</a>
                            <a href="./sub2_2.html">지난 익산식품대전</a>
                        </dd>
                    </dl>
                    <dl>
                        <dt><a href="./sub3.html">국가식품 기획전</a></dt>
                        <dd>
                            <a href="./sub3.html">참가기업 제품소개</a>
                        </dd>
                    </dl>
                    <dl>
                        <dt><a href="./sub4_1.html">국랜선 푸드어택</a></dt>
                        <dd>
                            <a href="./sub4_1.html">푸드어택 맛 보기</a>
                            <a href="./sub4_2.html">푸드어택 리뷰</a>
                        </dd>
                    </dl>
                    <dl>
                        <dt><a href="./sub5.html">이벤트</a></dt>
                        <dd>
                        </dd>
                    </dl>
                </div>
                <div class="menu_info">
                    <div class="menu_ct_01"><a href="#">한국식품산업클러스터진흥원</a></div>
                    <div class="menu_sns">
                        <a href="https://www.facebook.com/foodpolis.kr">
                            <!-- 페이스북 -->
                        </a>
                        <a href="https://www.instagram.com/foodpolis.kr/">
                            <!-- 인스타 -->
                        </a>
                        <a href="https://blog.naver.com/foodpolis1">
                            <!-- 네이버블로그 -->
                        </a>
                        <a href="https://www.youtube.com/channel/UCcdXmMG59sjCxLyINIlTW2Q/featured">
                            <!-- 유튜브 -->
                        </a>
                    </div>
                </div>
                <div class="all_menu_ft">
                    (54576) 전라북도 익산시 왕궁면 국가식품로 100<br>
                    한국식품산업클러스터진흥원 연락처 : 063-720-0500<br>
                    Copyright(c) 한국식품산업클러스터진흥원 All rights reserved.
                </div>
            </div>
        </div>
        <div class="bg"></div>
    </div>

    <!-- article -->
    <article>

    </article>

    <!-- section -->
    <div class="main">
        <div class="main_bnr main_section w_custom">
            <h3 class="main_title">주요프로그램</h3>
            <ul class="bnr_wrap clear">
                <li>
                    <a href="#"></a>
                    <div class="txt">
                        <p><strong>푸드어택 기획전</strong></p>
                        <p>푸드어택 기획전을<br>지금 바로 만나보세요</p>
                    </div>
                </li>
                <li>
                    <a href="#"></a>
                    <div class="txt">
                        <p><strong>온라인 홍보관</strong></p>
                        <p>국가식품클러스터의<br>유익한 정보를 파헤쳐보자!</p>
                    </div>
                </li>
                <li>
                    <a href="#"></a>
                    <div class="txt">
                        <p><strong>랜선 리얼 리뷰</strong></p>
                        <p>랜선으로 만나는<br>리얼한 리뷰를 확인하세요</p>
                    </div>
                </li>
                <li>
                    <a href="#"></a>
                    <div class="txt">
                        <p><strong>이벤트</strong></p>
                        <p>푸드어택기획전의<br>다양한 혜택을 누리세요</p>
                    </div>
                </li>
            </ul>
        </div>

        <div class="main_food main_section w_custom">
            <h3 class="main_title aos-init aos-animate" data-aos="fade-down" data-aos-offset="150" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out">푸드어택 맛 보기<img src="img/main/icon_youtube.png"></h3>
            <ul class="food_wrap clear aos-init aos-animate" data-aos="fade-up" data-aos-offset="0" data-aos-easing="ease-in-out">
                <li>
                    <div class="txt">
                        <p class="tit aos-init" data-aos="fade-down" data-aos-offset="150" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out"><strong>오세득의<br>국가식품클러스터<br>푸드어택</strong>
                        </p>
                        <p data-aos="fade-down" data-aos-offset="300" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out" class="aos-init">어서오세득</p>
                        <a href="#" target="_blank" class="link aos-init" data-aos="fade-down" data-aos-offset="350" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out">지금 만나러가기</a>
                    </div>
                    <div class="img_box aos-init" data-aos="fade-up" data-aos-offset="150" data-aos-delay="0" data-aos-duration="300">
                    </div>
                </li>
                <li>
                    <div class="txt">
                        <p class="tit aos-init" data-aos="fade-down" data-aos-offset="150" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out"><strong>다이어터를 위한<br>커피 마시는<br>6가지
                                방법</strong></p>
                        <p data-aos="fade-down" data-aos-offset="300" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out" class="aos-init">맛불리TV</p>
                        <a href="#t" target="_blank" class="link aos-init" data-aos="fade-down" data-aos-offset="350" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out">지금 만나러가기</a>
                    </div>
                    <div class="img_box aos-init" data-aos="fade-up" data-aos-offset="150" data-aos-delay="0" data-aos-duration="300">
                    </div>
                </li>
                <li>
                    <div class="txt">
                        <p class="tit aos-init" data-aos="fade-down" data-aos-offset="150" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out"><strong>유지어터<br>24시<br>브이로그</strong></p>
                        <p data-aos="fade-down" data-aos-offset="300" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out" class="aos-init">양수빈</p>
                        <a href="#" target="_blank" class="link aos-init" data-aos="fade-down" data-aos-offset="350" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out">지금 만나러가기</a>
                    </div>
                    <div class="img_box aos-init" data-aos="fade-up" data-aos-offset="150" data-aos-delay="0" data-aos-duration="300">
                    </div>
                </li>
            </ul>
        </div>

        <div class="main_event main_section">
            <h3 class="main_title aos-init" data-aos="fade-down" data-aos-offset="150" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out">푸드어택 싹쓰리</h3>
            <div class="event_wrap w_custom aos-init" data-aos-offset="150" data-aos="fade-up" data-aos-easing="ease-in-out">
                <ul class="swiper-wrapper">
                    <li class="swiper-slide">
                        <a href="#" alt="푸드어택 온라인 홍보관 관람인증 이벤트"></a>
                        <div class="thumb">
                            <img src="img/main/20201120132217_7326.jpg" alt="푸드어택 온라인 홍보480관 관람인증 이벤트">
                        </div>
                        <div class="txt">
                            <h4>푸드어택 온라인 홍보관 관람인증 이벤트</h4>
                            <p>마음에 드는 홍보관 콘텐츠 관람하고 인증하면 선물을 드려요</p>
                        </div>
                    </li>
                    <li class="swiper-slide">
                        <a href="#" alt="푸드어택 홈페이지 방문인증 이벤트"></a>
                        <div class="thumb">
                            <img src="img/main/20201120123330_4763.jpg" alt="푸드어택 홈페이지 방문인증 이벤트">
                        </div>
                        <div class="txt">
                            <h4>푸드어택 홈페이지 방문인증 이벤트</h4>
                            <p>국가식품클러스터 푸드어택 홈페이지 방문하고 선물받자</p>
                        </div>
                    </li>
                    <li class="swiper-slide">
                        <a href="#" alt="푸드어택 기획전 구매인증 이벤트"></a>
                        <div class="thumb">
                            <img src="img/main/20201120123143_7782.jpg" alt="푸드어택 기획전 구매인증 이벤트">
                        </div>
                        <div class="txt">
                            <h4>푸드어택 기획전 구매인증 이벤트</h4>
                            <p>국가식품클러스터 제품 구매하면 푸짐한 선물이!</p>
                        </div>
                    </li>
                </ul>

            </div>
        </div>


        <div class="main_insta main_section">
            <h3 class="main_title insta_title w_custom aos-init" data-aos="fade-down" data-aos-offset="200" data-aos-delay="0" data-aos-duration="500" data-aos-easing="ease-in-out">INSTAGRAM<span>#국가식품클러스터
                    #Foodpolis #푸드어택 #간편식품 #온라인기획전</span></h3>
            <div class="instar_wrap aos-init" data-aos="fade-up" data-aos-offset="150" data-aos-easing="ease-in-out">
                <div class="instar_box swiper-container-horizontal">
                    <ul id="instafeed" class="swiper-wrapper" style="transition-duration: 0ms; transform: translate3d(-1292.16px, 0px, 0px);">
                        <li class="swiper-slide swiper-slide-duplicate swiper-slide-duplicate-active" data-swiper-slide-index="0">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201120093817_7615.png" alt="인스타그램1120-1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate swiper-slide-duplicate-next" data-swiper-slide-index="1">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201130012745_7036.png" alt="인스타그램_1130_2_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate" data-swiper-slide-index="2">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201130012529_1539.png" alt="인스타그램_1130_1_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate" data-swiper-slide-index="3">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201126041356_2599.png" alt="인스타그램_1126_2_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate" data-swiper-slide-index="4">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201126040907_7171.png" alt="인스타그램_1126_1_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate swiper-slide-prev" data-swiper-slide-index="5">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201125013220_9154.png" alt="인스타그램_1125_2_1">
                        </li>
                        <li class="swiper-slide swiper-slide-active" data-swiper-slide-index="0">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201120093817_7615.png" alt="인스타그램1120-1">
                        </li>
                        <li class="swiper-slide swiper-slide-next" data-swiper-slide-index="1">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201130012745_7036.png" alt="인스타그램_1130_2_1">
                        </li>
                        <li class="swiper-slide" data-swiper-slide-index="2">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201130012529_1539.png" alt="인스타그램_1130_1_1">
                        </li>
                        <li class="swiper-slide" data-swiper-slide-index="3">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201126041356_2599.png" alt="인스타그램_1126_2_1">
                        </li>
                        <li class="swiper-slide" data-swiper-slide-index="4">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201126040907_7171.png" alt="인스타그램_1126_1_1">
                        </li>


                        <li class="swiper-slide swiper-slide-duplicate-prev" data-swiper-slide-index="5">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201125013220_9154.png" alt="인스타그램_1125_2_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate swiper-slide-duplicate-active" data-swiper-slide-index="0">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201120093817_7615.png" alt="인스타그램1120-1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate swiper-slide-duplicate-next" data-swiper-slide-index="1">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201130012745_7036.png" alt="인스타그램_1130_2_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate" data-swiper-slide-index="2">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201130012529_1539.png" alt="인스타그램_1130_1_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate" data-swiper-slide-index="3">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201126041356_2599.png" alt="인스타그램_1126_2_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate" data-swiper-slide-index="4">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201126040907_7171.png" alt="인스타그램_1126_1_1">
                        </li>
                        <li class="swiper-slide swiper-slide-duplicate" data-swiper-slide-index="5">
                            <a href="#" target="_blank"></a>
                            <img src="img/main/20201125013220_9154.png" alt="인스타그램_1125_2_1">
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        <div class="main_about main_section w_custom aos-init" data-aos="fade-up" data-aos-offset="200" data-aos-easing="ease-in-out">
            <h3><img src="img/main/ft_logo.png"></h3>
            <p>원료조달부터 시제품 생산, 검사 분석, 인력공급, 수출·마케팅까지<br>식품기업을 지원합니다</p>
        </div>
    </div>
    </div>

    <!-- footer -->
    <footer>
        <h2 class="hidden">푸터영역</h2>
        <div class="ft_inner">
            <ul>
                <li><a href="https://www.mafra.go.kr/sites/mafra/index.do"><img src="./img/main/partner_img01.png" alt="농림축산식품부"></a></li>
                <li><a href="https://www.jeonbuk.go.kr/"><img src="./img/main/partner_img02.png" alt="전라북도"></a>
                </li>
                <li><a href="http://www.iksan.go.kr/index.iksan?menuCd=DOM_000002017000000000"><img src="./img/main/partner_img03.png" alt="익산시"></a></li>
                <li><a href="https://www.foodpolis.kr/"><img src="./img/main/partner_img04.png" alt="푸드폴리스"></a>
                </li>
            </ul>
            <div class="info">
                <p>(54576) 전라북도 익산시 왕궁면 국가심풀로 100</p>
                <p>한국식품산업클러스터진흥원 연락처:063-720-0500</p>
                <p>Copyright&copy;한국식품산업클러스터진흥원 All rights reserved.</p>
            </div>
        </div>
    </footer>
</body>

</html>
