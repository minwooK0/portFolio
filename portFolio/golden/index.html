<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css"
        integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/Swiper/3.4.1/css/swiper.min.css">
    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300;400;500;700&display=swap"
        rel="stylesheet">
    <link rel="stylesheet" href="css/fuwat.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/hamberger.css">
    <title>골든 브라운</title>
</head>
<style>
.slide_show {
    height : 100vh;
    width : 100vw;
    z-index : 2000;
    position : absolute;
    background : black;
    animation : slideshow 1.5s forwards;
}

@keyframes slideshow{
    0% {top : 0;}
    50% {top : 0;}
    100% {top : -100vh;}

}
</style>
<body>

    <!-- main -->
    <header>
        <nav class="main_pc_nav_wrap">
            <ul>
                <a href="index.html">
                    <h2>GOLDEN BROWN</h2>
                </a>
                <li>
                    <a href="index.html">
                        Main
                        <span>Event&Topics</span>
                    </a>
                </li>
                <li>
                    <a href="sub01.html">
                        introduce
                        <span>About us</span>
                    </a>
                </li>
                <li>
                    <a href="sub02.html">
                       Menu
                        <span>Menu</span>
                    </a>
                </li>
           
        </nav>
        <nav class="sp_nav_wrap">
            <a href="index.html">
                <a href="#" class="btnMenu"><h2>GOLDEN BROWN</h2></a>
            </a>
            <div class="button">
                <a class="btn-open" href="#"></a>
            </div>
        </nav>
        <div class="overlay">
            <div class="wrap">
                <ul class="wrap-nav">
                    <li>
                        <p>Choice</p>
                        <ul>
                            <li>
                                <a href="index.html">
                                    Main
                                    <span>Event&Topics</span>
                                </a>
                            </li>
                            <li>
                                <a href="sub01.html">
                                    introduce
                                    <span>About us</span>
                                </a>
                            </li>
                            <li>
                                <a href="sub02.html">
                                   Menu
                                    <span>Menu</span>
                                </a>
                            </li>
                        </ul>
                    </li>
                    
                </ul>
                <small>&copy; Copyright. MINWOO KIM. All Rights Reserved.</small>
            </div>
        </div>
    </header>

    <section>
        <main>
            <p class="pcview">
                Professional<br />
                전문쉐프가 만드는 디저트！
            </p>
            <center>
                <h3>GOLDEN BROWN</h3>
            </center>
            <center>
                <div class="btn">
                    <a href="#">
                        Next Page
                    </a>
                </div>
            </center>

        </main>
        <!-- news -->
        <div class="content_wrap news fuwat">
            <center>
                <div class="main_title">
                    <h1>YOUR SPECIAL BAKERY PARTNER</h1>
                    <hr>
                    <span>Partner with your business</span>
                </div>
            </center>
            <div class="swiper-container">
                <div class="swiper-wrapper">
                    <div class="swiper-slide">
                        <a href="#">
                            <div class="news_title">
                                Bakery Partnership
                            </div>
                            <p class="news_day">first</p>
                            <p class="news_txt">골든 브라운은 높은 품질의 디저트 및 베이커리를 납품하여 누구나 카페를 더 쉽고 편하게 그리고 성공적으로
                                운영할 수 있도록 만들고자 합니다. 퀄리티 있는 제품들을 손쉽게 카페 메뉴화하여 손님들의 만족도를 높이고
                                매출을 끌어올릴 수 있습니다. 당신의 카페 비즈니스 성공을 위해 우리의 베이커리 파트너십을 믿고
                                맡겨보세요.</p>
                        </a>
                    </div>
                    <div class="swiper-slide">
                        <a href="#">
                            <div class="news_title">
                                More Story about us
                            </div>
                            <p class="news_day">second</p>
                            <p class="news_txt">지난 2012 년부터 시작된 골든 브라운의 더 많은 이야기와 우리가 지키고자 하는 디저트의
                                가치에 대한 이야기가 궁금하신가요?</p>
                        </a>
                    </div>
                    <div class="swiper-slide">
                        <a href="#">
                            <div class="news_title">
                                Products of Quality
                            </div>
                            <p class="news_day">third</p>
                            <p class="news_txt">골든 브라운은 좋은 품질과 맛에 대한 엄격한 기준을 가지고,
                                디저트 및 베이커리 본연의 맛과 식감을 충분히 느끼면서 즐길 수 있는 제품들을
                                생산하고 제안합니다.</p>
                        </a>
                    </div>
                </div>
            </div>
        </div>

        <!-- about -->
        <div class="content_wrap about fuwat">
            <div class="main_title">
                <h1>Menu</h1>
                <hr>
                <span>About us</span>
            </div>
            <div class="row about_img">
                <div class="col-md-6">
                    <img src="img/blog_dalgona.png" alt="음식">
                    <h1>달고나</h1>
                    <p>부드럽고 향긋한 달고나. 클래식한 메뉴부터 모던한 느낌의 메뉴까지 다양한 컨셉의 달고나를 만나보세요.</p>
                </div>
                <div class="col-md-6">
                    <img src="img/blog_dirty.jpg" alt="음식">
                    <h1>디저트</h1>
                    <p>브런치부터 달콤한 디저트빵까지 골든 브라운의 다양한 종류의 빵들이 준비되어 있습니다.</p>
                </div>
                <div class="col-md-6">
                    <img src="img/burnt-cheese-instagram.jpg" alt="음식">
                    <h1>치즈</h1>
                    <p>다양한 토핑, 포장재 등 카페 운영에 필요한 다양한 부자재들을 준비하였습니다.</p>
                </div>
                <div class="col-md-6">
                    <img src="img/opera-cake-instagram.jpg" alt="음식">
                    <h1>케이크</h1>
                    <p>부드럽고 향긋한 케이크. 클래식한 메뉴부터 모던한 느낌의 메뉴까지 다양한 컨셉의 케이크를 만나보세요.</p>
                </div>
            </div>
        </div>

        <!-- message -->
        <div class="content_wrap message fuwat">
            <div class="main_title">
                <h1>순서</h1>
                <hr>
                <span>Message</span>
            </div>
            <center>
                <h1>제품을 납품 받고 싶으시다면?</h1>
                <p>
                    골든 브라운의 제품을 납품 받고 싶으시다면 아래의 절차를 거쳐 신청해주시기 바랍니다.
                </p>
            </center>
            <div class="container">
                <div class="row">
                    <div class="col-md-3"  id="step">
                        <dl style="color: aliceblue;justify-content: center; justify-content: center;">
                            <dt>Step 1</dt>
                            <img src="img/step1.svg"  style="background-color:thistle;">
                            <dd>신규 거래 신청서 작성<br />
                                신규 거래를 위한<br />
                                신청서 양식을 작성해주세요.</dd>
                        </dl>
                    </div>
                    <div class="col-md-3"  id="step">
                        <dl style="color: aliceblue;">
                            <dt>Step 2</dt>
                            <img src="img/step2.svg"  style="background-color:thistle;">
                            <dd>첨부파일<br />
                                신청서 작성 시 첨부하지 못한<br />
                                사업자등록증/통장사본을<br />
                                팩스로 접수해주세요.</dd>
                        </dl>
                    </div>
                    <div class="col-md-3"  id="step">
                        <dl style="color: aliceblue;">
                            <dt>Step 3</dt>
                            <img src="img/step3.svg"  style="background-color:thistle;">
                            <dd>상담 및 안내<br />
                                본사 상담원이 고객님께<br />
                                이메일/카카오톡으로<br />
                                가입 안내를 드립니다.</dd>
                        </dl>
                    </div>
                    <div class="col-md-3"  id="step">
                        <dl style="color: aliceblue;">
                            <dt>Step 4</dt>
                            <img src="img/step4.svg"  style="background-color:thistle;">
                            <dd>온라인 주문에 필요한<br />
                                아이디와 비밀번호를<br />
                                발급해드립니다.</dd>
                        </dl>
                    </div>
                    </div>
                </div>
            </div>
        </div>
        </div>

        <!-- menu -->
        <div class="content_wrap menu fuwat">
            <div class="main_title">
                <center>
                    <h1>Main</h1>
                    <hr>
                    <span>Menu</span>
                    <p class="spview">메인 메뉴입니다。</p>
                </center>

                <div class="row menu_wrap">
                    <div class="col-md-4">
                        <a href="#">
                            <div class="menu_img">
                                <img src="img/insta_caramel.jpg">
                            
                            </div>
                        </a>
                    </div>
                    <div class="col-md-4">
                        <a href="#">
                            <div class="menu_img">
                                <img src="img/insta_malcha.jpg">
                               
                            </div>
                        </a>
                    </div>
                    <div class="col-md-4">
                        <a href="#">
                            <div class="menu_img">
                                <img src="img/burnt-cheese-instagram.jpg">
                              
                            </div>
                        </a>
                    </div>
                    <p class="menu_txt">
                        여러가지 제품을 시식해보세요。
                    </p>


                    <center>
                        <div class="btn">
                            <a href="#">바로가기</a>
                        </div>
                    </center>
                </div>
            </div>
        </div>

        <div class="content_wrap about fuwat">
			<div class="container mt-3">
                <center>
				<h2>Video</h2>
						<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
				
					<div class="carousel-inner">
						<div class="carousel-item active">
                            <iframe width="70%" height="315" src="https://www.youtube.com/embed/w4qEb_-lMXs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
						</div>
                    </center>
		</div>


        <!-- contact -->
        <div class="content_wrap contact fuwat">
            <div class="main_title">
                <center>
                    <h1>궁금하신점이 있으신가요?</h1>
                    <hr>
                    <span>Contact</span>
                </center>

                <div class="row contact_wrap">
                    <div class="input_wrap">
                        <label>이름</label><span>*</span></label>
                        <input type="text"><br />
                    </div>
                    <div class="input_wrap">
                        <label>이메일<span>*</span></label>
                        <input type="email"><br />
                    </div>
                    <div class="input_wrap">
                        <label>제목</label>
                        <input type="tel"><br />
                    </div>
                    <div class="input_wrap">
                        <label>문의사항</label>
                        <textarea></textarea>
                    </div>

                    <center>
                        <div class="btn">
                            <a href="#">
                                SEND
                            </a>
                        </div>
                    </center>
                </div>
            </div>
        </div>
        <footer>
            <div class="row">
                <div class="col-md-6 pcview">
                    <ul>
                        <li><a href="#">엔와이푸드</a></li>
                        <li><a href="#">사업자 등록번호 ***-***-****</a></li>
                        <li><a href="#">경기도 부천시   </a></li>
                        <li><a href="#">1588-****</a></li>
                        <li><a href="#">예금주 ***</a></li>
                    </ul>
                </div>
                <div class="col-md-6 faq_wrap pcview">
                    <a href="#">고객센터</a>
                    <div class="btn">
                        <a href="#">
                          1566-****
                        </a>
                    </div>
                </div>
                <div class="col-md-5 faq_wrap spview">
                    <a href="#">ご予約・お問合わせは</a><br />
                    <div class="btn">
                        <a href="#">
                            03-6240-0471
                        </a>
                    </div>
                </div>
                <div class="col-md-8 pcview">
                    <small>&copy; Copyright. MINWOO KIM. All Rights Reserved.</small>
                </div>
                <div class="col-md-4 sns_wrap" style="margin-top:40px;">
                    <a href="#">
                        <img src="img/sns_ico01.png" alt="sns">
                    </a>
                    <a href="#">
                        <img src="img/sns_ico02.png" alt="sns">
                    </a>
                    <a href="#">
                        <img src="img/sns_ico03.png" alt="sns">
                    </a>
                    <a href="#">
                        <img src="img/sns_ico04.png" alt="sns">
                    </a>
                </div>
                <div class="privacy">
                    <a href="#">PORTFOLIO</a>
                </div>
            </div>
        </footer>
    </section>





    <script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js'>
        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js"
        integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI"
        crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/Swiper/3.4.1/js/swiper.min.js"></script>
    <script>
        var mySwiper = new Swiper ('.swiper-container', {
            loop: true,
        slidesPerView: 2,
        spaceBetween: 90,
        centeredSlides : true,
        pagination: '.swiper-pagination',
        nextButton: '.swiper-button-next',
        prevButton: '.swiper-button-prev',
        breakpoints: {
            767: {
            slidesPerView: 1,
        spaceBetween: 0
            }
            }
        })
    </script>
    <script src="js/hamberger.js"></script>
    <script src="js/fuwat.js"></script>
</body>

</html>